gruntfile.js

```diff
@@ -3,30 +3,53 @@ module.exports = function(grunt) {
   // Project configuration.
   grunt.initConfig({
     pkg: grunt.file.readJSON('package.json'),
+
     sass: {
-      options: {
-        sourceMap: true
-        },
       dist: {
-        files: {
-          'assets/css/app.css' : 'assets/sass/app.scss'
-        }
+        files: [{
+          expand: true,
+          cwd: 'www/sass',
```

Comment:
@Neb19  you should put all .css .scss and .js files in a directory like assets/css or assets/js ... instead putting them directly in www/